### PR TITLE
import flutter_localizations for example plug&play

### PR DESCRIPTION
**Pull request by oguzozgen**
README updated.
import flutter_localizations for example plug&play

On example plug&play action, 
      --> GlobalMaterialLocalizations.delegate 
      --> GlobalWidgetsLocalizations.delegate 
      --> GlobalCupertinoLocalizations.delegate 
classes requires flutter_localizations import.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,23 @@ I made This plugin because google deprecated [Place Picker](https://developers.g
...
 
+Pubspec changes:
+
+```
+      dependencies: 
+      
+        google_maps_flutter: ^0.5.30
+        google_map_location_picker: ^3.3.4
+        flutter_localizations:
+          sdk: flutter
```
Comment (humazed) [inline README.md:25]: `flutter_localizations` are not required to use this plugin why did you add it to the readme?

Comment (oguzozgen) [reply README.md:25]: When copy pasted the sample, flutter says down sided classes are missing. I checked your repo and i saw that dependency decleration on imports and I did same. I solved the missing classes. And it was the only missing classes for copy paste and run.


--> GlobalMaterialLocalizations.delegate
--> GlobalWidgetsLocalizations.delegate
--> GlobalCupertinoLocalizations.delegate
